recipes/taskipy/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "taskipy" %}
+{% set version = "1.10.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0006429f708f530fc7add28ca51fd41f6c6e1fbe86763ff05125e1af3e8bdc7e
+
+build:
+  skip: true  # [win]
```

Comment:
This looks like a noarch package. What is the reason to skip Windows?

Reply:
I believe the reasoning is given here: https://github.com/conda-forge/staged-recipes/pull/22761#issuecomment-1536787747

The project uses `mslex` instead of `shlex` for Windows, and that isn't available on Conda Forge: https://github.com/taskipy/taskipy/blob/master/taskipy/task_runner.py#L18

Reply:
If it is only availability, we still prefer noarch. The package won't be installable on windows anyway and, when the missing dependency is available, we don't need to do anything here. It will be installable already.

Reply:
@ocefpaf @eugenetriguba 

Thanks for the help!